src/Compilers/VisualBasic/Portable/Scanner/CharacterInfo.vb

```diff
@@ -16,6 +16,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
     ''' </summary>
     Partial Public Class SyntaxFacts
 
+        Friend Shared Function GetUnicodeCategory(ch As Char) As UnicodeCategory
+            If ch = ChrW(&H2028) Then
+                ' U+2028 "LINE SEPARATOR" should be classified by CharUnicodeInfo.GetUnicodeCategory as
+                ' UnicodeCategory.LineSeparator but due to a bug Is incorrectly categorized as
+                ' UnicodeCategory.Format. See https://github.com/dotnet/coreclr/issues/3542
+                Return UnicodeCategory.LineSeparator
+            End If
+
+            Return CharUnicodeInfo.GetUnicodeCategory(ch)
+        End Function
```

Comment:
Is this method necessary in addition to the version in `UnicodeCharacterUtilities`?